src/librustdoc/visit_lib.rs

```diff
@@ -25,7 +25,7 @@ impl<'a, 'tcx> LibEmbargoVisitor<'a, 'tcx> {
     crate fn new(cx: &'a mut crate::core::DocContext<'tcx>) -> LibEmbargoVisitor<'a, 'tcx> {
         LibEmbargoVisitor {
```

Comment:
Might I say, `LibEmbargoVisitor` is one of the most interesting names I've seen in `rust-lang/rust`.